From 5f5b1c6d522bdc3ae18af13b97c40d3cbfc5ca9f Mon Sep 17 00:00:00 2001
From: Nina Gertskin <ngertskin@habana.ai>
Date: Sun, 09 Aug 2020 10:41:42 +0300
Subject: [PATCH] [SW-16836] embedded: home directory as parameter

use default home directory if the environment varialbe is not defined
or not set before calling to make

Change-Id: Icd874de12086efa34148e0c534b59c1ec5d900e0
Signed-off-by: Nina Gertskin <ngertskin@habana.ai>
---

diff --git a/package/robot/robot.mk b/package/robot/robot.mk
index 92e32bc..f43a00f 100644
--- a/package/robot/robot.mk
+++ b/package/robot/robot.mk
@@ -5,9 +5,13 @@
 $(eval $(generic-package))
 
 LTP_TESTSUITE = 0
-
 ifeq ($(RUN_LTP_TESTSUITE),yes)
 	LTP_TESTSUITE = 1
+endif
+
+HOME_DIR = "/home"
+ifneq ($(HOMEDIR),)
+        HOME_DIR = $(HOMEDIR)
 endif
 
 define ROBOT_BUILD_CMDS
@@ -19,7 +23,7 @@
 			-v PDU_PORT:$(PDU_PORT)				\
 			-v SERIAL_PORT:$(SERIAL_PORT)			\
 			-v DEVICE_NAME:$(HL_DEV)			\
-			-v HOMEDIR:$(HOMEDIR)				\
+			-v HOMEDIR:$(HOME_DIR)				\
 			run_robot_regression.robot			\
 	)
 endef
